Tiltfile

```diff
@@ -394,15 +378,11 @@ def prepare_all():
                 name = name,
                 context = context,
             )
```

Comment:
Do we actually need lines 367 to 380 anymore? The `kustomive_build_arg` and `providers_arg` are not used anymore and looks like providers and kustomize are handled in `tilit-prepare` directly.

Reply:
yes missed it after the rebase removed them